### PR TITLE
chore: Setup jest and enzyme

### LINKED ISSUE #38: Setup Jest & Enzyme

### PR DESCRIPTION
**Pull request by amitbadala**
Resolve #38 

Client Test using Shallow and Jest
clientJestTest

Server test using Jest
serverJestTest

### CODE REVIEW

File: server/routes/boards.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import http from 'http';
+import server from './boards';
+import { PORT } from '../configs/config';
+
+describe('httpServer', () => {
```
Comment (nimish-gupta) [inline server/routes/boards.test.js]: how are you starting the server? Check [this](https://github.com/visionmedia/supertest) for alternate option.

Comment (amitbadala) [reply server/routes/boards.test.js]: Done !!

File: .gitignore [hunk 2]
```diff
@@ -17,6 +17,7 @@ client/build
 .env.test.local
 .env.production.local
 .env
+.rej
```
Comment (nimish-gupta) [inline .gitignore]: what are these files?

Comment (amitbadala) [reply .gitignore]: 1. We might create .env files for environment variables for production, testing environments. These files should not be committed into code.
2. .rej are reject files when we apply patch.

**Pull request merged**